quail/processes/wps_climdex_su.py

```diff
@@ -0,0 +1,140 @@
+from rpy2 import robjects
+from pywps import Process, LiteralInput, ComplexInput, ComplexOutput, FORMATS
+from pywps.app.Common import Metadata
+from pywps.inout.formats import Format
+from tempfile import NamedTemporaryFile
+
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
+from wps_tools.io import log_level
+from quail.utils import get_package, logger
+
+
+class ClimdexSU(Process):
+    """
+    Takes a climdexInput object as input and computes the SU
+    (summer days) climdexindex:  that is,  the annual count of days where
+    daily maximum temperature exceeds 25 degreesCelsius
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "build_rda": 90,
+            },
+        )
+        inputs = [
+            ComplexInput(
+                "climdex_input",
+                "climdexInput",
+                abstract="R Object of type climdexInput",
+                supported_formats=[
+                    Format("application/x-gzip", extension=".rda", encoding="base64")
```

Comment:
What actually happened when you used `application/rda`? Did `birdy` reject the input?

Reply:
`application/rda` is read as text instead of binary. That link I sent you shows which mime types decode as binary. When I looked at the file type of a rdata files it says `gzip` so that's why I used this one from the available list

Reply:
also this only happened on docker and not locally